### ISSUE TITLE
Greenworks Addon/Plugin not working

### ISSUE THREAD

**Issue by asdzxc8** [Scirra/Construct-3-bugs#3022]
## Problem description

Both achievements and overlay not working for all the platforms (Win, Mac and Linux).
Implemented the same way I did 9 months back with almost no deviations but somehow even the Win version are not working after publishing live this time round.

Tried using 0.30.4 (which was working last year - at least for Win platform), and 0.30.3, 0.30.3-r2 but to no avail.
Exported with r148 (stable) and r152 (beta).

## Attach a .c3p

I'm willing to provide any project files/exported file for examination or Steam game code through email to the person assigned instead of putting the file up online if that's alright. Please kindly let me know which one you want.

## Steps to reproduce

1. Followed the instructions by exporting the project file NW.js: 0.33.3 when trying out both the 0.14; NW.js: 0.30.4 for 0.13. 
What is different this time round is I uncheck "Package Assets", "Compress final zip", "Window frame" and "resizable frame" but I've also tried another version checking them all like before.
2. Added the 3 files (dll and txt) respectively to their folder.
3. Uploaded to Steamworks on my Macbook and set default build, then publish live.

## Observed result

1. Clicking on Achievement (from Main Menu > More) should activate Overlay but nothing comes out instead.
2. Upon achievement unlock, there are no visual popup on the bottom right side of the screen to inform the players.
3. Steam doesn't record the Achievement into their system as well.
*Previously this work perfectly for Win, no visual popup for Mac but Steam still record the achievements. This time round nothing at all.

## Expected result

1. Expect Overlay to work for at least Win (doesn't work on both Mac and Linux before).
2. Expect visual popup to inform players at the bottom right side of the screen for Win platform at least (doesn't work on both Mac and Linux before).
3) Expect achievements to record on Steam for all 3 platforms (Win, Mac and Linux). Working on all 3 platforms previously.

## More details

**Affected browsers/platforms:** 
Affected on both Win64 (Win 10) and Mac64 (macOS)

**First affected release:** 
Exported with r148 (stable) first and later r152 (beta).
Have not tried with r111 (stable - which I used 9 months back).

## System details

<details><summary>View details</summary>

NIL
Will provide any screenshot or video upon request.

</details>

**Comment by AshleyScirra** [comment]
I can guarantee you the 0.30.4 version of Greenworks has not changed since its release. So if it used to work but doesn't any more, this suggests it wasn't Greenworks that broke.

You have to match the exact versions of Greenworks, NW.js, and the Steamworks SDK. You have not mentioned which Steamworks SDK version you were using, and you did mention that you tried unsupported versions of NW.js like 0.30.3 that are definitely not supported, so I suspect this is a version mismatch problem, perhaps with the Steamworks SDK version.

For us to further investigate please follow all the bug report guidelines including providing a minimal project - see the guidelines regarding the fact we don't want your whole project, and how to derive a minimal project.

**Comment by asdzxc8** [comment]
@AshleyScirra The only constant thing between 9 months back and now is the Steamworks SDK (1.42). No reason to mention it since I'm replicating exactly what I'm doing back then.

Also I did not tried unsupported version of NW.js 0.30.3 like you suggested. I'm going to assume you speed read and misunderstood somehow.

9 months back when I exported the project, there are Greenworks files in the folder; "greenworks.js", "greenworks-win64.node", and so on.
Is there a reason why I'm not seeing them anymore this time round?
Added with the facts that Nw.js exporting were down for days last week. Is this a coincidence or am I reading too much into it?
